### PR TITLE
 UI/TTL helperEnabled/DisabledText fix 

### PR DESCRIPTION
**Pull request by hellobontempo**
**The UI defaults to generate credentials' TTL and Max-TTL for a role respectively to expire after 1 hr and 24hrs respectively, so this update is to clarify this behavior to customers using the UI by explicitly stating these expiry times.**

 **Old Role Text:**
Screen Shot 2021-07-30 at 2 33 35 PM

---

**New Role Text:**
Screen Shot 2021-07-30 at 2 30 49 PM

---

**Old Rotation TTL Text:**
_toggled off_
Screen Shot 2021-07-30 at 2 33 54 PM

---

_toggled on_
Screen Shot 2021-07-30 at 2 34 09 PM

---

**Corrected Text:**
_toggled off_
Screen Shot 2021-07-30 at 2 31 37 PM

_toggled on_
Screen Shot 2021-07-30 at 2 32 02 PM

### CODE REVIEW

File: ui/app/models/database/role.js [hunk 1]
```diff
@@ -45,8 +45,9 @@ export default Model.extend({
   rotation_period: attr({
     editType: 'ttl',
     defaultValue: '24h',
-    subText:
+    helperTextDisabled:
```
Comment (hashishaw) [inline ui/app/models/database/role.js:48]: Do any other models' attributes with `editType: 'ttl',` have `subtext`? If so we should switch them out

File: ui/app/models/database/role.js [hunk 2]
```diff
@@ -29,14 +29,14 @@ export default Model.extend({
     editType: 'ttl',
     defaultValue: '1h',
     label: 'Generated credentials’s Time-to-Live (TTL)',
-    subText: 'Vault will use the engine default of 1 hour',
+    helperTextDisabled: 'Vault will use a TTL of 1 hour.',
```
Comment (Monkeychip) [inline ui/app/models/database/role.js:32]: Did Ivana have any thoughts on helperTextEnabled for these?

File: ui/app/models/database/role.js (hunk 1 above)
Comment (hellobontempo) [reply ui/app/models/database/role.js:48]: Nope these were the only TTL cases of `subText`. 

`aws-credential.js` does have a `helpText` attribute but I was planning to address that in another PR so this change could be merged. I was planning to set up AWS in order to test that TTL

File: ui/app/models/database/role.js (hunk 2 above)
Comment (hellobontempo) [reply ui/app/models/database/role.js:32]: No, but it'd be good to clarify a final time when she is back tomorrow.

Comment (hellobontempo) [reply ui/app/models/database/role.js:32]: @Monkeychip - Ivana wants to stick with the default text!

**Review by Monkeychip — APPROVED** [review]
Nice work!

**Pull request merged**